libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -69,33 +74,42 @@ class AC_WPNav
 
     /// set current target climb or descent rate during wp navigation
     void set_speed_up_cms(float speed_up_cms);
+    void set_speed_up_ms(float speed_up_ms);
     void set_speed_down_cms(float speed_down_cms);
+    void set_speed_down_ms(float speed_down_ms);
 
     /// get default target horizontal velocity during wp navigation
-    float get_default_speed_NE_cms() const { return _wp_speed_cms; }
+    float get_default_speed_NE_cms() const { return get_default_speed_NE_ms() * 100.0; }
```

Comment:
this is OK but maybe we should just leave, "get_default_speed_NE_cms" as returning "_wp_speed_cms" instead of going through the multiply and divide by 100 each time.

Reply:
No, we need to ensure that all data goes though the meters version of the function or we may find one of them is broken when we remove the cm.